std/internal/math/biguintcore.d

```diff
@@ -108,12 +108,12 @@ private:
         opAssign(x);
     }
 
-    enum trustedAssumeUnique = function(BigDigit[] input) pure @trusted {
+    enum trustedAssumeUnique = function(BigDigit[] input) pure @trusted @nogc {
         return assumeUnique(input);
```

Comment:
Isn't this an abuse of `@trusted`, per the recent forum discussions? We are relying on the caller to make sure that the argument is, in fact, unique, but what if it isn't?